### PR TITLE
JBPM-8057: Process instance log details panel with unique id

### PR DESCRIPTION
**Pull request by nmirasch**
Use the item view hascode to generate details panel unique id and

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @jsoltes while process instance admin operations retriggering  testing I found this issue when the date didn't have enough precision to differentiate between logs of the same node the 'Details' link don't work propertly with duplicated ids. In this PR I changed that id generation using the hash code. Could you take a look? Thanks!

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogItemView.java [hunk 1]
```diff
@@ -133,7 +133,10 @@ public void setValue(final ProcessInstanceLogSummary model) {
         if (model.getWorkItemId() == null) {
             detailsPanelDiv.setHidden(true);
         } else {
-            setDetailsPanelAttributes(model);
+            String panelId = "_" + this.hashCode();
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogItemView.java]:
```suggestion
            String panelId = Document.get().createUniqueId()
```

Comment (nmirasch) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogItemView.java]: yep, looks better id generation, Thanks!

**Comment by nmirasch** [comment]
@cristianonicolai  updated!

**Comment by nmirasch** [comment]
@jsoltes makes sense for me remove the test. Let me remove if from the PR. Thanks!

**Comment by nmirasch** [comment]
@jsoltes done

**Pull request merged**